index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+
+// Create PDO connection (inc set fetch mode)
+$db = new PDO ('mysql:host=db; dbname=milo-collection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+// Write SQL query to retrieve all collection items
+$query = $db->prepare("SELECT `name`, `age`, `instrument`, `band`, `technical-prowess`, `image` FROM `idols`;");
+$query->execute();
+$idols = $query->fetchAll();
+
+// Loop over results to echo out stats for each item
+function displayItems($idols) {
```

Comment:
This doesn't necessarily need to be in a function. If you're going to call it in your code more than once, then it absolutely should be in a function, but here I think we only need to iterate over the array once. Maybe worth a discussion?